### PR TITLE
Fix several spelling corrections + refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8977,6 +8980,7 @@ degress->degrees, digress,
 deimiter->delimiter
 deine->define
 deined->denied, defined,
+deines->denies, defines, defined,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8983]: I don't think the last one is very likely here, unless you've seen it in the wild?
```suggestion
deines->denies, defines,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -17508,6 +17520,7 @@ isntallation->installation
 isntallations->installations
 isntance->instance
 isntances->instances
+isntead->instead
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:17523]: Maybe `isn't read` too?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -17575,6 +17588,8 @@ itslev->itself
 itteration->iteration
 itterations->iterations
 iunior->junior
+iverse->diverse, inverse,
+iversed->inverse
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:17592]: This says this correction is also valid, but probably rarer:
https://www.dictionary.com/browse/inversed
```suggestion
iversed->inverse, inversed,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -19213,6 +19228,7 @@ modulues->modules
 modyfy->modify
 moent->moment
 moeny->money
+moew->more
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:19231]: More variants:
```suggestion
moew->more, meow, mow,
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -22946,6 +22963,7 @@ proffesional->professional
 proffesor->professor
 proffessor->professor
 profilic->prolific
+profirle->profile
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22966]: More versions:
```suggestion
profirle->profile
profirled->profiled
profirler->profiler
profirles->profiles
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -24806,7 +24825,9 @@ remved->removed
 remves->removes
 remvoe->remove
 remvoed->removed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24827]:
```suggestion
remvoed->removed
remvoes->removes
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -30355,7 +30376,7 @@ travesed->traversed
 traveses->traverses
 travesing->traversing
 tre->tree
-treadet->treated
+treadet->treated, treaded,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:30379]: Possibly threaded too, but might be a stretch.

**Review by peternewman — CHANGES_REQUESTED** [review]
Some improvements

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:8983]: Yes, I've seen it. Not sure where. But the key proximity of the `s` and `d` are to blame i would guess

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 972a761

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 972a761

**Comment by luzpaz** [comment]
Ok, checks passed. I'll rebase now.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:8993]: Fair enough.

**Review by peternewman — APPROVED** [review]

**Pull request merged**